.chloggen/mx-psi_hostmetadata-gzip.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Shouldn't this be a bug_fix?

---

I wasn't sure, since it actually worked fine before, but I guess if you have a proxy in between things could break.
```suggestion
change_type: bug_fix
```